docker-dist/Dockerfile

```diff
@@ -19,29 +19,22 @@ FROM jboss/wildfly:10.1.0.Final
 
 MAINTAINER Heiko W. Rupp <hawkular-dev@lists.jboss.org>
 
-ADD target/installer.jar /tmp/
 # ADD test-simple.war /opt/jboss/wildfly/standalone/deployments/
-ADD src/main/resources/standalone.conf /opt/jboss/wildfly/bin
+ADD target/hawkular-javaagent-wildfly-dist/bin/standalone.conf $JBOSS_HOME/bin
+ADD target/hawkular-javaagent-wildfly-dist/bin/hawkular-javaagent.jar $JBOSS_HOME/bin
+ADD target/hawkular-javaagent-wildfly-dist/standalone/configuration/hawkular-javaagent-config.yaml $JBOSS_HOME/standalone/configuration/
 
+ENV HAWKULAR_SERVER_PROTOCOL http
 ENV HAWKULAR_SERVER_ADDR hawkular
 ENV HAWKULAR_SERVER_PORT 8080
 ENV HAWKULAR_AGENT_USER=jdoe HAWKULAR_AGENT_PASSWORD=password
-ENV JAVA_OPTS="-Xmx256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true -XX:MaxMetaspaceSize=256m"
+ENV JAVA_OPTS="-Xmx256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman,org.jboss.logmanager -Djava.awt.headless=true -XX:MaxMetaspaceSize=256m"
```

Comment:
Is included [here](https://github.com/hawkular/hawkular-agent/pull/313/files/149416054c57bceffd9b2fca21227bce7d62d421#diff-f235893104df7b8b55fa6ebf8e49c6d1R23) in the custom [standalone.conf](https://github.com/hawkular/hawkular-agent/blob/master/hawkular-javaagent-wildfly-feature-pack/src/main/resources/bin/standalone-hawkular-javaagent-javaopts.conf#L18-L21) from the feature-pack.